website/content/en/blog/log-namespacing.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Log Namespacing
+short: Log Namespacing
+description: Changing Vector's data model
+authors: ["fuchsnj"]
+date: "2023-??-??"
+badges:
+  type: announcement
+  domains: ["log namespacing"]
+tags: []
+---
+
+The Vector team has been hard at work improving the data model of events in Vector. These
+changes are now available for beta testing for those who want to try it out and give feedback.
+This is an opt-in feature. Nothing should change unless you specifically enable it.
+
+## Why
+
+Currently, all data for events is placed at the root of the event, regardless of where the data came
+from or how it was obtained. Not only can that make it confusing to understand what a certain field
+represents (eg: was the `timestamp` field generated by Vector when it was ingested, or is it when
+the source originally created the event) but it can easily cause data collisions.
+
+Log namespacing also unblocks powerful features being worked on, such as end-to-end type checking
+of events in Vector.
+
+
+## How to enable
+
+The [global config] `schema.log_namespace` can be set to `true` to enable the new
+Log Namespacing feature for all components. The default is `false`.
+
+Every source also has a `log_namespace` config option. This will override the global setting,
+so you can try out Log Namespacing on individual sources.
+
+The following example enables the `log_namespace` feature globally, then disables it for a single
+source.
+
+```toml
+schema.log_namespace = true
+
+[sources.input_with_log_namespace]
+type = "demo_logs"
+format = "shuffle"
+lines = ["input_with_log_namespace"]
+interval = 1
+
+[sources.input_without_log_namespace]
+type = "demo_logs"
+format = "shuffle"
+lines = ["input_without_log_namespace"]
+interval = 1
+log_namespace = false
+
+[sinks.console]
+type = "console"
+inputs = ["input_with_log_namespace", "input_without_log_namespace"]
+encoding.codec = "json"
+
+```
+
+## How It Works
+
+
+### Data Layout
+
+There are three distinct types of data that Vector handles.
+
+(Examples are from the `http_client` source)
```

Comment:
Given the following examples are from `datadog_agent` does it make more sense for this example to also be from the same?